lib/java/gradlew.bat

```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off
```

Comment:
This script is autogenerated by the Gradle Wrapper integration. It is expected to exist and is updated as Gradle Wrapper is refreshed to a newer version.